circle.yml

```diff
@@ -19,7 +19,7 @@ checkout:
   post:
     - git pull origin ${CIRCLE_BRANCH}
     - git clone git@github.com:wholesale-design-system/styleguide.git
-    - git config --global user.email circleci@circleci
+    - git config --global user.email t.schlage@gmx.net
```

Comment:
As I said you can create such a user if you want :D
I don't know where to store the credentials in a secure way so that also peoples after us get the access.